abb_experimental.rosinstall

```diff
@@ -0,0 +1,5 @@
+#Temporary until PR accepted by https://github.com/ros-industrial/
+- git: {local-name: abb, uri: 'https://github.com/AustinDeric/abb.git', version: kinetic-devel}
+
+- git: {local-name: industrial_core, uri: 'https://github.com/ros-industrial/industrial_core.git', version: kinetic-devel}
```

Comment:
>  So in effect we will have to do rosinstall + rosdep rather than just rosinstall

Doing a `rosdep install ..` after a `wstool merge ..` (which I think you meant) is regular development practice I think?